src/script/lua_api/l_util.cpp

```diff
@@ -357,6 +357,22 @@ int ModApiUtil::l_get_dir_list(lua_State *L)
 	return 1;
 }
 
+// safe_file_write(path, content)
+int ModApiUtil::l_safe_file_write(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	const char *path = luaL_checkstring(L, 1);
+	const char *content = luaL_checkstring(L, 2);
```

Comment:
Why limit to regular strings only? Modifying binary data should be allowed too IMO.

Reply:
didn't even consider that, will add